npm/ng-packs/packages/schematics/src/utils/text.ts

```diff
@@ -10,6 +10,9 @@ export const macro = (text: string) => upper(snake(text));
 export const dir = (text: string) =>
   strings.dasherize(text.replace(/\./g, '/').replace(/\/\//g, '/'));
 
+// TODO: Is there a standard function to do this? There is a leak if the value contains single quote.
+export const quote = (value: number | string) => typeof value === 'string' ? `'${value}'` : value;
```

Comment:
Is there a standard function to do this?

There is a leak if the value contains single quote, although enum won't contains symbol.